hudi-common/src/main/avro/HoodieRollbackMetadata.avsc

```diff
@@ -31,18 +31,22 @@
             {"name": "partitionPath", "type": "string"},
             {"name": "successDeleteFiles", "type": {"type": "array", "items": "string"}},
             {"name": "failedDeleteFiles", "type": {"type": "array", "items": "string"}},
-            {"name": "appendFiles", "type": {
+            {"name": "rollbackLogFiles", "type": {
                 "type": "map",
-                "doc": "Files to which append blocks were written",
                 "values": {
                     "type": "long",
                     "doc": "Size of this file in bytes"
                 }
+            }},
+            {"name": "writtenLogFiles", "type": {
```

Comment:
why do we call this `writtenLogFiles` and elsewhere we just call this `probable...`?